Lab3-Node.js-Basics/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "lab3-node.js-basics",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/node": "^20.2.1",
+    "ts-node": "^10.9.1",
+    "typescript": "^5.0.4"
+  },
+  "dependencies": {
+    "axios": "^1.4.0",
```

Comment:
this packet not used at backend, you can use undici (it's included in node 20) or undici.fetch if you need fetch interface